reporting_app/templates/untabbed_datatables.html

```diff
@@ -7,8 +7,11 @@
 {% endblock %}
 
 {% block content %}
-
-{{ description }}
+{% if description_html %}
```

Comment:
This would be a lot cleaner in a separate template using `{% include %}`, but can refactor that later